modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/model/ReferralTrafficChannelImpl.java

```diff
@@ -154,7 +155,11 @@ private JSONArray _getReferringDomainsJSONArray() {
 		Stream<ReferringURL> stream = _referringDomains.stream();
 
 		return JSONUtil.putAll(
-			stream.map(
+			stream.limit(
+				10
```

Comment:
Should this limit be configurable?

Reply:
Good question. I don't think it's very valuable at this point but it could be in the future. I'm going to write it down to take a look because I'm not sure how to do it technically (_configuration in plain classes in Liferay_). Thanks, Cris!

Reply:
Could you ask UX and PM, if this value could be configurable?

Reply:
Sure!

Reply:
https://issues.liferay.com/browse/LPS-113917?focusedCommentId=2332338&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-2332338

Reply:
It should NOT for now ;)

https://issues.liferay.com/browse/LPS-113917?focusedCommentId=2332342&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-2332342